samples/DurableFunctions/README.md

```diff
@@ -0,0 +1,326 @@
+ 
+
+## Run your Durable Functions app on Azure 
+
+Before you can deploy your container to your Azure Container apps you need to create an Container Apps environment with a Log Analytics workspace and a Azure Functions app. 
+
+1\. Login to your Azure subscription
+
+```sh 
+az login
+  
+az account set -subscription | -s <subscription_name>
+
+az upgrade
+
+az extension add --name containerapp --upgrade
+
+az provider register --namespace Microsoft.Web
+
+az provider register --namespace Microsoft.App
+
+az provider register --namespace Microsoft.OperationalInsights
+```
+---
+
+2\. Create azure container app environment
+
+Create an environment with an auto-generated Log Analytics workspace.
+
+```sh
+  az group create --name MyResourceGroup --location northeurope
+  az containerapp env create -n MyContainerappEnvironment -g MyResourceGroup --location northeurope
+  ```
+
+4\. Create the function app
+ 
+ Run the [az functionapp create](https://learn.microsoft.com/en-us/cli/azure/functionapp#az-functionapp-create) command to create a new function app in the new managed environment backed by azure container apps.
+
+```sh
+az functionapp create --resource-group MyResourceGroup --name <functionapp_name> \
+--environment MyContainerappEnvironment \
+--storage-account <Storage_name> \
+--functions-version 4 \
+--runtime dotnet-isolated \
+--image <DOCKER_ID>/<image_name>:<version> 
+```
+
+In this example, replace **MyContainerappEnvironment** with the Azure container apps environment name. Also, replace <STORAGE_NAME> with the name of the account you used in the previous step, <APP_NAME> with a globally unique name appropriate to you, and <DOCKER_ID> or <login-server> with your Docker Hub ID.
+
+By this point, you should see the following resources created in "MyResourceGroup" on the Azure portal:
+ - Storage account
+ - SQL database
+ - Container app environment 
+ - Functions app
+ - Application Insights 
+
+
+5\. Set required app settings
+
+Get you Azure Storage and Azure SQL connection strings. Then go to your function app on portal, expand **Settings**, then click **Environment variables**. Add `SQLDB_Connection` app settings and set its value to the SQL connection strings, then click Apply. The `AzureWebJobsStorage` setting should already be there - check that it's set to your storage account's connection string. Click Apply again and confirm to update the settings. 
+
+6\. Start your Durable Functions orchestration
+
+Go to the URL associated with the HTTP trigger to start the orchestration. To do that, find the URL of your app on Azure Portal (look on the top right of the **Essentials** section in the **Overview** tab), it should look something like the following:
+
+```
+https://<< your app name >>.northeurope.azurecontainerapps.io
+```
+
+Add the following to the end of the URL:
+
+```
+/api/{your http trigger name}
+```
+
+In the sample code above, the name is `DurableFunctionsOrchestrationCSharp_HttpStart`. 
```

Comment:
nit: I'd also include the explicit URL generated by this app, so `/api/DurableFunctionsOrchestrationCSharp_HttpStart`. That'll make copy-pasting to the browser easier